Source/WebCore/css/CSSProperties.json

```diff
@@ -4977,8 +4977,8 @@
             "codegen-properties": {
                 "initial": "initialZeroLength",
                 "name-for-methods": "StrokeDashOffset",
-                "converter": "Length",
-                "parser-grammar": "<length-percentage svg>"
+                "converter": "LengthAllowingNumber",
+                "parser-grammar": "<length-percentage> | <number>"
```

Comment:
Since this matches the SVG2 draft definition, perhaps we should update the specification link to that:

https://svgwg.org/svg2-draft/painting.html#StrokeDashoffsetProperty?

Reply:
Sure, I can update to the SVG2 spec link. It is unfortunate the the fill-stroke-3 spec is not in agreement though :(
The good thing is that this change is aligning us with Chrome too, not just the SVG2 spec. The bad thing is that it seems Firefox  still matches our old behavior :/